bentoml/_internal/yatai_rest_api_client/config.py

```diff
@@ -26,6 +26,8 @@ class YataiClientContext:
     name: str
     endpoint: str
     api_token: str
+    email: str
+    yatai_version: str
```

Comment:
We don't need to track yatai version here, since we are already tracking it on the yatai side.

Also, as Yatai version continue increases, we will have to keep checking the version to the latest.  

My suggestion is not track yatai version in bentoml

Reply:
can we exported to an envars, like `YATAI_VERSION` so that we can just retrieved it on bentoml side